.circleci/config.yml

```diff
@@ -194,6 +194,7 @@ references:
     run:
       name: Validate Files and Yaml
       when: always
+      no_output_timeout: 1h
```

Comment:
This change was merged in a different PR, pull from master

Reply:
ok